editor/editor_node.cpp

```diff
@@ -2179,6 +2180,15 @@ void EditorNode::_dialog_action(String p_file) {
 
 		} break;
 
+		case FILE_PACK_PROJECT_AS_ZIP: {
+			ProjectZIPPacker::pack_project_zip(p_file);
+			{
+				Ref<FileAccess> f = FileAccess::open(p_file, FileAccess::READ);
+				ERR_FAIL_COND_MSG(f.is_null(), "Unable to create ZIP file.");
```

Comment:
Print the attempted output file path:

```suggestion
				ERR_FAIL_COND_MSG(f.is_null(), vformat("Unable to create ZIP file at: %s", p_file));
```